data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -508,6 +508,16 @@
       [ "lye_powder", 100 ]
     ]
   ]
+},{
+  "type" : "recipe",
+  "result": "mutagen_jabberblood", "//": "jabberwocks have a few of these pumping in them, and they can be crushed down for a lot of mutagen",
```

Comment:
Based on your comment here and also from testing, I think the recipe could use a tool of some kind, probably a mortar and pestle maybe? Or maybe even quality hammering or something (fine cutting, cutting)?